etc/bash_completion.d/brew

```diff
@@ -52,8 +53,8 @@ __brew_complete_formulae() {
 
 __brew_complete_installed() {
   local cur="${COMP_WORDS[COMP_CWORD]}"
-  local inst="$(ls "$(brew --cellar)")"
-  COMPREPLY=($(compgen -W "$inst" -- "$cur"))
+  local inst=("$(brew --cellar)"/*)
+  COMPREPLY=($(compgen -W "${inst[*]#$(brew --cellar)/}" -- "$cur"))
```

Comment:
Suppose, in a dystopian future where packages with spaces in their names exist, wouldn't `inst[@]` be more correct than `inst[*]` here?